PyInstaller/hooks/hook-django.py

```diff
@@ -90,14 +112,16 @@
     installed_apps = eval(get_module_attribute(package_name + '.settings', 'INSTALLED_APPS'))
     migration_modules.extend(set(app + '.migrations' for app in installed_apps))
     # Copy migration files.
-    for mod in migration_modules:
-        mod_name, bundle_name = mod.split('.', 1)
-        mod_dir = os.path.dirname(get_module_file_attribute(mod_name))
-        bundle_dir = bundle_name.replace('.', os.sep)
-        pattern = os.path.join(mod_dir, bundle_dir, '*.py')
-        files = glob.glob(pattern)
-        for f in files:
-            datas.append((f, os.path.join(mod_name, bundle_dir)))
+    datas += collect_modules_files(migration_modules)
+
+    # Bundle django command scripts as data files.
+    # They are necessary get commands working.
+    logger.info('Collecting Django commands scripts.')
+    commands_modules = ['django.core.management.commands']
+    commands_modules.extend(set(app + '.management.commands' for app in installed_apps))
+    # Copy commands files.
+    datas += collect_modules_files(commands_modules)
```

Comment:
I assume, this change is solving some other [django related issues](https://github.com/pyinstaller/pyinstaller/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+django), eg.g #2313, and is unrelated to Django 2.0. So it should go into a separate commit.